compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -761,6 +763,40 @@ static LogicalResult setTransformDialectConfig(func::FuncOp entryPoint,
   return setTranslationInfo(entryPoint, translationInfo);
 }
 
+static bool isMatvecLike(linalg::LinalgOp linalgOp) {
+  if (linalgOp.getNumDpsInputs() != 2)
+    return false;
```

Comment:
[optional] `inferContractionDims` already checks for 2 inputs, so you can drop this check. I don't mind if you want to keep it for clarity though.